src/main/kotlin/stringcalculator/Application.kt

```diff
@@ -0,0 +1,47 @@
+package stringcalculator
+
+import stringcalculator.domain.Calculator
+import stringcalculator.ui.InputView
+
+fun main() {
+    Application().run()
+}
+
+private const val INDEX_OF_NUMBER = 0
+
+private const val INDEX_OF_SIGN = 1
+
+class Application {
+
+    fun run() {
+        val inputFormula = InputView().inputFormula()
+        val result = calculate(inputFormula)
+        print(result)
+    }
+
+    private fun calculate(inputFormula: List<String>): Int {
```

Comment:
calculate를 하는 역할은 Calculator 객체가 하는 것이 맞지 않을까요? 🤔

Reply:
메서드 파라미터로 전달되는 값이 `입력된` **식**인지 알 필요가 없다고 생각해요. 😃 

```suggestion
    private fun calculate(formula: List<String>): Int {
```